src/app/_components/HeadingWithIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import { Heading } from '@/components/Heading'
+
+export function HeadingWithIcon({
+  icon,
+  children,
+}: {
+  icon: React.ReactNode
+  children: React.ReactNode
```

Comment:
```suggestion
  children: string
```

Reply:
Don't hesitate to change what was previously written if you think it should be something else.